pkg/repository/manifest/repo.go

```diff
@@ -16,16 +16,32 @@ package manifest
 import (
 	"encoding/json"
 	"fmt"
-	"io"
+	"io/ioutil"
 	"time"
 
-	cjson "github.com/gibson042/canonicaljson-go"
 	"github.com/pingcap-incubator/tiup/pkg/repository/crypto"
-	"github.com/pingcap/errors"
 )
 
-// Init creates and initializes an empty reposityro
-func Init(dst string, initTime time.Time) error {
+// Init creates and initializes an empty repository
+func Init(dst string, initTime time.Time, pub, priv string) error {
+	// read key files
+	privBytes, err := ioutil.ReadFile(priv)
+	if err != nil {
+		return err
+	}
+	privKey := &crypto.RSAPrivKey{}
+	if err = privKey.Deserialize(privBytes); err != nil {
+		return err
+	}
+	pubBytes, err := ioutil.ReadFile(pub)
```

Comment:
Why do we need to store the public key? we can obtain the public key from the private key directly.